src/input_common/mouse/mouse_input.cpp

```diff
@@ -143,6 +143,15 @@ void Mouse::ReleaseButton(MouseButton button_) {
     mouse_info[button_index].data.axis = {0, 0};
 }
 
+void Mouse::ReleaseAllButtons() {
+    buttons = 0;
+    for (std::size_t button_index = 0; button_index < mouse_info.size(); button_index++) {
```

Comment:
This seems like it can be compressed into a ranged for loop like so:
```suggestion
    for (auto& button : mouse_info) {
```